website/contributing/release-faq.md

```diff
@@ -29,7 +29,7 @@ We follow a release cycle that is not strictly monthly - you can read more [abou
 
 ### How do I know if my fix/feature is in a certain release?
 
-To determine whether a fix or feature is present in a given release, you will need the commit hash where the fix or feature was added to the `main` branch of the core `react-native` repo. If you know the PR, you can look for the comment from **@facebook-github-bot** that says 'closed this in <COMMIT_HASH>'.
+To determine whether a fix or feature is present in a given release, you will need the commit hash where the fix or feature was added to the `main` branch of the core `react-native` repo. If you know the PR, you can look for the comment from **@facebook-github-bot** that says `closed this in <COMMIT_HASH>`.
```

Comment:
is this change ok? We can find another alternative like 

```
'closed this in `<COMMIT_HASH>`'
```

Reply:
yeah, I prefer this other as "closed this in" is not code.

Reply:
I applied @Simek suggestion for consistency:

![CleanShot 2023-08-03 at 19 44 53@2x](https://github.com/facebook/react-native-website/assets/749374/bf5d9b1a-e569-41a5-945e-cbc364b3e62b)

Can we keep it or do you prefer an inline code block?